haskell-tests/Types/PayloadSerializationSpec.hs

```diff
@@ -7,256 +7,31 @@ import Test.QuickCheck.Monadic
-        if nextIndex == 255 then
-          return (KeyIndex nextIndex:is)
-        else go (KeyIndex nextIndex:is) (nextIndex+1) (n-1)
-  reverse <$> go [] 0 maxLen
-
-genAccountKeysMap :: Gen (Map.Map KeyIndex VerifyKey)
-genAccountKeysMap = do
-  indexList <- genIndices
-  mapList <- forM indexList $ \idx -> do
-    kp <- genSigSchemeKeyPair
-    return (idx, correspondingVerifyKey kp)
-  return $ Map.fromList mapList
-
-genCredentialPublicKeys :: Gen CredentialPublicKeys
-genCredentialPublicKeys = do
-  credKeys <- genAccountKeysMap
-  credThreshold <- genSignThreshold
-  return CredentialPublicKeys{..}
-
-genCredentialDeploymentInformation :: Gen CredentialDeploymentInformation
-genCredentialDeploymentInformation = do
-  cdvPublicKeys <- genCredentialPublicKeys
-  cdvCredId <- RegIdCred . generateGroupElementFromSeed globalContext <$> arbitrary
-  cdvIpId <- IP_ID <$> arbitrary
-  cdvArData <- Map.fromList <$> listOf (do
-    ardName <- do
-      n <- arbitrary
-      if n == 0 then return (ArIdentity 1) else return (ArIdentity n)
-    ardIdCredPubShare <- AREnc <$> genElgamalCipher
-    return (ardName, ChainArData{..}))
-  cdvThreshold <- Threshold <$> choose (1, max 1 (fromIntegral (length cdvArData)))
-  cdvPolicy <- do
-    let ym = YearMonth <$> choose (1000,9999) <*> choose (1,12)
-    pValidTo <- ym
-    pCreatedAt <- ym
-    let pItems = Map.empty
-    return Policy{..}
-  cdiProofs <- do l <- choose (0, 10000)
-                  Proofs . BSS.pack <$> vector l
-  let cdiValues = CredentialDeploymentValues{..}
-  return CredentialDeploymentInformation{..}
 
+import Generators
 
 testSerializeEncryptedTransfer :: Property
-testSerializeEncryptedTransfer = property $ \gen gen1 seed1 seed2 -> forAll genAddress $ \addr -> monadicIO $ do
-  let public = AccountEncryptionKey . deriveElgamalPublicKey globalContext . generateGroupElementFromSeed globalContext $ seed1
-  let private = generateElgamalSecretKeyFromSeed globalContext seed2
-  let agg = makeAggregatedDecryptedAmount (encryptAmountZeroRandomness globalContext gen) gen (EncryptedAmountAggIndex gen1)
-  let amount = gen `div` 2
-  Just eatd <- run (makeEncryptedAmountTransferData globalContext (_elgamalPublicKey public) private agg amount)
-  return (checkPayload SP1 (EncryptedAmountTransfer addr eatd))
+testSerializeEncryptedTransfer =
+  property $ \gen gen1 seed1 seed2 -> forAll genAccountAddress $ \addr -> monadicIO $ do
+    let public = AccountEncryptionKey . deriveElgamalPublicKey globalContext . generateGroupElementFromSeed globalContext $ seed1
+    let private = generateElgamalSecretKeyFromSeed globalContext seed2
+    let agg = makeAggregatedDecryptedAmount (encryptAmountZeroRandomness globalContext gen) gen (EncryptedAmountAggIndex gen1)
+    let amount = gen `div` 2
+    Just eatd <- run (makeEncryptedAmountTransferData globalContext (_elgamalPublicKey public) private agg amount)
+    return (checkPayload SP1 (EncryptedAmountTransfer addr eatd))
```

Comment:
Please refrain from needless formatting. there is no need to spend time reviewing whether anything has changed or not.